### ISSUE TITLE
getCreativeCategory() doesn't work correctly

### ISSUE THREAD

**Issue by IDev-mc** [PaperSpigot/Paper#8770]
### Expected behavior

Returns the correct CreativeCategory from a Material

### Observed/Actual behavior

Allways returns BUILDING_BLOCKS

### Steps/models to reproduce

get the CreativeCategory from a Material and check if its the right one

### Plugin and Datapack List

FFA(The problematic Plugin), LuckPerms, Multiverse-Core, Plan, WebConsole, Worldedit

### Paper version

paper-1.19.3-371

**Comment by Machine-Maker** [comment]
Yeah, there’s an open PR to fix this. The current method will be deprecated and replaced because items can be in multiple categories now.
